api4/license.go

```diff
@@ -296,3 +299,123 @@ func getPrevTrialLicense(c *Context, w http.ResponseWriter, r *http.Request) {
 
 	w.Write([]byte(model.MapToJSON(clientLicense)))
 }
+
+func requestTrueUpReview(c *Context, w http.ResponseWriter, r *http.Request) {
+	// Only admins can request a true up review.
+	if !c.App.SessionHasPermissionTo(*c.AppContext.Session(), model.PermissionManageSystem) {
+		c.SetPermissionError(model.PermissionManageLicenseInformation)
+		return
+	}
+
+	license := c.App.Channels().License()
+	if license == nil {
+		http.Error(w, "A License is required to perform a true-up review", http.StatusBadRequest)
+		return
+	}
+
+	var subscription *model.Subscription
+	var err error
+	if c.App.Cloud() != nil {
```

Comment:
Good point, I think I may have been missing some context here on who actually needs to use this feature. It makes sense that it would be self-hosted only. I'll add the NotImplemented error in the case of cloud users.